scripts/backfill_cdc.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
```

Comment:
@armenzg Is this a reasonable place for a script that people may want/need to run on demand on the dev environment?

Reply:
@fpacifici that's a reasonable place since it's for development.

Reply:
@armenzg I think this needs approval from owners-sentry-dev